### ISSUE TITLE
Test the creation of fake student accounts in a batch using coldfront

### ISSUE THREAD

**Issue by DanNiESh** [nerc-project/operations#259]
We should test fake account creation before students begin using NERC to identify any issues related to rbac roles and cluster access.

**Comment by knikolla** [comment]
Is this for the production ColdFront and Keycloak? This warrants a larger discussion related to fake users in such systems.

We might want to create a separate realm in Keycloak for these users, integrate OpenShift to this test realm, etc.

**Comment by knikolla** [comment]
My advice to proceed quickly is

1. Create a new test realm in the NERC Keycloak
2. Integrate this new realm into the already existing ColdFront staging environment, setting up authentication through it. As we have already integrated this with the production realm it's only a matter of switching the URL and some parameters.
3. Integrate this realm as a new alternative log in option for the NERC OCP, so that users in this test realm are able to authenticate to OCP. As we have already integrated the production realm, it's only a matter of duplicating the config entry and shifting some parameters.
4. Use the Keycloak REST API to create the desired number of test users in this new realm. These users will then show up in the ColdFront staging web interface when a PI searches for users to add to a resource allocation and will be able to log in to OCP.
5. Set up the desired test scenario by creating resource allocations and assigning users to these projects. I can help writing simple scripts to facilitate this.

**Comment by joachimweyl** [comment]
blocked by https://github.com/nerc-project/operations/issues/274

**Comment by DanNiESh** [comment]
blocked by https://github.com/nerc-project/operations/issues/190

**Comment by joachimweyl** [comment]
@DanNiESh This is no longer blocked. Is this something you see finishing this sprint or shall we push it out to the next since the blockers did not get unblocked until the end of the sprint?

**Comment by DanNiESh** [comment]
I think it's no longer needed since we've tested with real user email addresses.

**Comment by joachimweyl** [comment]
closing as it is no longer needed.
